libs/platform-server/package.json

```diff
@@ -5,9 +5,8 @@
     "@angular/common": ">12.2.0",
     "@angular/core": ">12.2.0",
     "@angular/platform-server": ">12.2.0",
-    "@scullyio/ng-lib": "x.x.x"
+    "@scullyio/ng-lib": "x.x.x",
```

Comment:
Is this for any version? I think it is best to add a version here .

Reply:
we should make an todo for this. For now its ok, but you are not wring. However, the version should really matter, as long as its there.